docs/src/main/markdown/customizations.md

```diff
@@ -78,6 +79,9 @@ enums to a single Scala file.
 
 - Typically, enum values appear in UPPER_CASE in proto files, and ScalaPB generates case objects
   with exactly the same name in Scala. If you would like ScalaPB to transform the names into CamelCase, set `enum_value_naming` to `CAMEL_CASE`.
+  
+- To avoid setting scala name for each enum value, you can instead set `enum_strip_prefix` to `true`. In this case prefix if it contains enum name plus optional underscore or upper-case enum nape with optional underscore for enum values will be stripped.
```

Comment:
- It is a common practice in protobufs to prefix each enum value name with the name of the enum. For example, an enum name `Size` may have values named `SIZE_SMALL` and `SIZE_LARGE`. When you set `enum_strip_prefix` to true, ScalaPB will strip the enum's name from each value name, and they would become `SMALL` and `LARGE`. Then the name can be transformed to camel-case according to `enum_value_naming`. Note that the prefix that is removed is the all-caps version of the enum name followed by an underscore.